app/services/surf_app_sign_up_service.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class SurfAppSignUpService < AppSignUpService
+  def call(app, remote_ip, params)
```

Comment:
We can probably remove this entire method since it should be the same as what's in the base class.